packages/edit-navigation/src/store/actions.js

```diff
@@ -164,129 +161,3 @@ function* batchSave( menuId, menuItemsByClientId, navigationBlock ) {
 		body,
 	} );
 }
-
-function computeCustomizedAttribute( blocks, menuId, menuItemsByClientId ) {
```

Comment:
Moved this function into `utils` so that we can better test it in isolation without exporting it as an action.